projects/commudle-admin/src/app/feature-modules/labs/components/labs/search-bar/search-bar.component.ts

```diff
@@ -7,9 +10,59 @@ import { Component, OnInit } from '@angular/core';
 })
 export class SearchBarComponent implements OnInit {
 
-  constructor() { }
+  filterLabItems = [
+    {title: 'Popular Labs'},
+    {title: 'New Labs'},
+    {title: 'Labs by Experts'}
+  ];
```

Comment:
we can actually write it as:
filterLabItems = {
  popular: 'Popular Labs',
 new: 'New Labs'
...
}

and use the keyvalue pipe for iterating over it (I've used it at a couple of places in the code itself)

Reply:
Initial work is done